core/assets/bundles/bundle.properties

```diff
@@ -1314,4 +1315,26 @@ block.spectre.description = A massive dual-barreled cannon. Shoots large armor-p
 block.meltdown.description = A massive laser cannon. Charges and fires a persistent laser beam at nearby enemies. Requires coolant to operate.
 block.repair-point.description = Continuously heals the closest damaged unit in its vicinity.
 block.segment.description = Damages and destroys incoming projectiles. Laser projectiles are not targeted.
-
+block.resupply-point.description = Refills the ammo supply of units using power.
+block.parallax.description = Fires a tractor beam at air units, making it more difficult for them to fly away.
+block.foreshadow.description = Targets the enemy with the highest health and inflicts massive damage to them.
+block.ground-factory.description = Produces basic ground units.
+block.air-factory.description = Produces basic air units.
+block.naval-factory.description = Produces basic aquatic units.
+block.additive-reconstructor.description = Reconstructs basic units into more advanced units.
+block.multiplicative-reconstructor.description = Reconstructs units from an Additive Reconstructor into more advanced units.
+block.exponential-reconstructor.description = Reconstructs units from a Multiplicative Reconstructor into even more advanced units.
+block.tetrative-reconstructor.description = Reconstructs units from an Exponential Reconstructor into the most advanced units possible.
+block.payload-conveyor.description = Allows for transport of small blocks or units.
+block.payload-router.description = Splits payloads to up to three other directions equally.
+block.disassembler.description = A slag processing structure that can condense thorium from slag. It is less efficient overall, due to requiring raw scrap as well.
```

Comment:
this entire line might need to be rephrased, seems to explain to much vs whats normal for descriptions